programs/voter-stake-registry/Cargo.toml

```diff
@@ -36,16 +36,16 @@ bytemuck = "1.9.1"
 #          Recently the discriminator for new VoterWeightRecord accounts has changed, and upgrading
 #          this dependency here without also upgrading the spl-governance program instance beforehand
 #          would lead to VWR accounts that are unusable until the spl-governance program is upgraded.
-spl-governance = { version = "=2.2.1", features = ["no-entrypoint"] }
-spl-governance-addin-api = "=0.1.1"
+spl-governance = { version = "=3.1.1", features = ["no-entrypoint"] }
```

Comment:
this needs to be checked out in relation to the warning above - likely all gov deployments have updated, but need to double check